src/App.tsx

```diff
@@ -32,78 +85,53 @@ export const App: React.FC = () => {
         </label>
       </div>
 
-      {/* Move this markup to Pagination */}
       <ul className="pagination">
-        <li className="page-item disabled">
+        <li className={cn('page-item', { disabled: currentPage === 1 })}>
           <a
             data-cy="prevLink"
             className="page-link"
             href="#prev"
-            aria-disabled="true">
+            aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
`===` always returns a Boolean type. Will be the same. Use it for all such cases
```suggestion
            aria-disabled={currentPage === 1}
```